lms/resources/_js_config/file_picker_config.py

```diff
@@ -31,6 +33,9 @@ def blackboard_config(cls, context, request, application_instance):
     def canvas_config(cls, context, request, application_instance):
         """Get Canvas files config."""
 
+        if not context.is_canvas:
+            return {}
```

Comment:
Related to https://github.com/hypothesis/lms/pull/3356 but if returning None here it breaks this branch.